docs/changelog/README.rst

```diff
@@ -0,0 +1,18 @@
+.. examples for changelog entries adding to your Pull Requests
+
+file ``544.doc.rst``::
+
+    Update requirements file to replace
+    requirements.txt with requirements-build.in - by :user:`akasurde`
+
+file ``544.feature.rst``::
+
+    Added sftp get functionality to fetch file from remote host  - by :user:`amolkahat`
+
+file ``571.bugfix.rst``::
+
+    Fix double close issue, ddd logic to free
+    the channel allocated memory within
+    `__dealloc__`  - by :user:`ganeshrn`
+
+.. see pyproject.toml for all available categories  (tool.towncrier.type)
```

Comment:
This is a code comment, it won't show up in the render but it should (as README files show up in GitHub UI when the folder view is open)